R/crmPack-package.R

```diff
@@ -28,20 +28,20 @@
 #  cpp_coxfit
 #' @import checkmate
 #' @import ggplot2
-#' @import methods 
+#' @import methods
 #' @importFrom gridExtra arrangeGrob
 #' @importFrom graphics plot hist legend lines matlines matplot
 #' @importFrom stats binomial coef cov2cor gaussian glm lm median model.matrix
 #' optim pgamma plogis pnorm qgamma qlogis qnorm quantile rbinom rgamma approxfun
-#' rnorm runif uniroot var vcov step mad
+#' rnorm runif uniroot var vcov step mad pbeta
```

Comment:
note: ok, but not necessary. preferred way is to use `stats::pbeta` in the code

Reply:
Will do as you prefer. Note: pbeta is also used in the for the related stopping rule.